src/components/ChatApp/Settings/AccountTab.react.js

```diff
@@ -108,17 +259,70 @@ class AccountTab extends React.Component {
     }
   };
 
+  handleAvatarTypeChange = event => {
+    this.setState({
+      avatarType: event.target.value,
+      settingsChanged: true,
+      isAvatarAdded: false,
+      file: '',
+      avatarSrc: '',
+    });
+  };
+
+  handleAvatarSubmit = () => {
+    const { file } = this.state;
+    const { accessToken } = this.props;
+    // eslint-disable-next-line no-undef
+    let form = new FormData();
+    form.append('access_token', accessToken);
+    form.append('image', file);
+    this.setState({ uploadingAvatar: true });
+    setUserAvatar(form).then(() => {
+      this.setState({
+        uploadingAvatar: false,
+        isAvatarAdded: true,
+        isAvatarUploaded: true,
+      });
```

Comment:
We again need to call the getAvatar action to see the change reflected on the StaticBar